filesystem/fat32/fat32_test.go

```diff
@@ -258,7 +264,7 @@ func TestFat32Read(t *testing.T) {
 				corrupted := ""
 				if tt.bytechange >= 0 {
 					b := make([]byte, 1)
-					_, _ = rand.Read(b)
+					_, _ = chacha.Read(b)
```

Comment:
Any reason to switch away from `crypto/rand` to `math/rand/v2` ?
https://pkg.go.dev/math/rand/v2#ChaCha8.Read is not available for Go 1.22
- https://github.com/diskfs/go-diskfs/pull/262

Reply:
Nope. It was done because lint was complaining when go1.23. With 1.22 we should not switch.